src/translations/en.json

```diff
@@ -2478,7 +2478,7 @@
             "triggers": {
               "name": "Triggers",
               "header": "When",
-              "description": "This list of triggers is what starts your automation. A trigger is a specific event happening in or around your home, for example: ''When the sun sets''.",
+              "description": "This list of triggers is what starts your automation, and any listed trigger can start it. A trigger is a specific event happening in or around your home, for example: ''When the sun sets''.",
```

Comment:
I checked with our technical writer @c0ffeeca7 
We do not want to make these sentences any longer than what they already are.
We would be ok to change the whole sentence to:

> A trigger is a specific event happening in or around your home, for example: ''When the sun sets''. Any trigger listed here will start your automations

Reply:
That’s what I had in mind.

Reply:
Thank you, @jlpouffier ! I've put that sentence by the technical writer in.

Maybe nit-picky, but wouldn't it be better if it was singular "automation" at the end of that sentence given that the user interface provides a way to edit only one automation at a time?